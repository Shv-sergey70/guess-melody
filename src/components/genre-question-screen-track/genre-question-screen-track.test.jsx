import React from 'react';
import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import GenreQuestionScreenTrack from "./genre-question-screen-track";

configure({adapter: new Adapter()});

test(`GenreQuestionScreenTrack correctly renders`, () => {
  const tree = shallow(
      <GenreQuestionScreenTrack
        trackIndex={1}
        isSelected={true}
        onChange={jest.fn()} >
        Custom children. Audio Player
      </GenreQuestionScreenTrack>
  );

  expect(tree).toMatchSnapshot();
});
