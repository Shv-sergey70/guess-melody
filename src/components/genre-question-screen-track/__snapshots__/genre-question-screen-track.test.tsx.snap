// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreQuestionScreenTrack correctly renders 1`] = `ShallowWrapper {}`;
